roles/lgsm/defaults/main.yml

```diff
@@ -0,0 +1,130 @@
+########################################################################
+# Title:         Sandbox: LGSM | Default Variables                     #
+# Author(s):     Scarabyte                                             #
+# URL:           https://github.com/saltyorg/Sandbox                   #
+# --                                                                   #
+########################################################################
+#                   GNU General Public License v3.0                    #
+########################################################################
+---
+################################
+# Basics
+################################
+
+lgsm_instances: ["lgsm"]
```

Comment:
We should default this to blank since configuration is required or have a sample commented and point to docs.

Reply:
```yaml
lgsm_instances: []
``` 
would suffice